client/webpack.config.js

```diff
@@ -3,6 +3,21 @@ const HtmlWebPackPlugin = require("html-webpack-plugin");
 
 module.exports = {
   entry: './index.js',
+  devServer: {
```

Comment:
devServer config should be part of a `webpack.development.config.js` file. This should be used for prod builds